secret_santa.py

```diff
@@ -105,30 +106,30 @@ def main(argv=None):
         participants = config['PARTICIPANTS']
         if len(participants) < 2:
             raise Exception('Not enough participants specified.')
-        
+
         givers = []
         for person in participants:
-            name, email, amazon = re.match(r'([^<]*)<([^>]*)>.*(http.*)', person).groups()
+            name, email, amazon, country = re.match(r'([^<]*)<([^>]*)>.*(http.*) ([A-z]{2})', person).groups()
             name = name.strip()
```

Comment:
It is, but the data is coming from somewhere else, a user (especially one who isn't overly technical) won't be like...oh yeah, belgium is country code BE.  Let me write that in.

The data is coming in from:  https://osalt.tech/events/santa.  You could also have some helpful mappers.  ie.  "United States" ==> "United States of America" or some variation of that for common alternate spellings.